vars/fcosKolaTestIso.groovy

```diff
@@ -0,0 +1,34 @@
+// Run kola testiso
+// Available parameters:
+//     cosaDir:        string  -- cosa working directory
+//     extraArgs:      string  -- extra arguments to pass to `kola testiso`
+//     extraArgs4k:    string  -- extra arguments to pass to 4k `kola testiso`
+//     skipMetal4k:    boolean -- skip metal4k image
+def call(params = [:]) {
+    def cosaDir = utils.getCosaDir(params)
+    def extraArgs = params.get('extraArgs', "");
+    def extraArgs4k = params.get('extraArgs4k', "");
+
+    testIsoRuns = [:]
+    testIsoRuns["Metal"] = {
```

Comment:
Super minor: let's call the branches `metal` and `metal4k`? That matches the lowercase of `fcosKola`'s `run` and `run-upgrade`.